src/Microsoft.ML.Recommender/doc.xml

```diff
@@ -0,0 +1,90 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<doc>
+  <members>
+    
+    <member name="MatrixFactorizationTrainer">
+      <summary>
+        Train a matrix factorization model using <a href='https://www.csie.ntu.edu.tw/~cjlin/libmf/'>LIBMF.
```

Comment:
>. [](start = 106, length = 1)

</a>

Reply:
Any idea of previewing this file locally, I don't know how Microsoft's doc site will process it.

---
In reply to: [229411844](https://github.com/dotnet/machinelearning/pull/1450#discussion_r229411844) [](ancestors = 229411844)

Reply:
Thanks. As I move my doc in code, the complier now automatically checks those naive errors.

---
In reply to: [229418963](https://github.com/dotnet/machinelearning/pull/1450#discussion_r229418963) [](ancestors = 229418963,229411844)